### ISSUE TITLE
Devices with all-katakana names are presented as nameless

### ISSUE THREAD

**Issue by colons** [appcelerator/node-ios-device#3]
My iPhone is called 'チルノ'.

```
$ cat detect.js 
var iosDevice = require('node-ios-device');

iosDevice.devices(function (err, devices) {
    console.log(devices);
});

$ node detect.js    
[ { udid: '[…]',
    buildVersion: '12F70',
    cpuArchitecture: 'arm64',
    deviceClass: 'iPhone',
    deviceColor: '#e1e4e3',
    hardwareModel: 'N61AP',
    modelNumber: 'MG482',
    productType: 'iPhone7,2',
    productVersion: '8.3',
    serialNumber: '[…]' } ]
```

But when I rename the device to 'cirno':

```
$ node detect.js
[…]
    name: 'cirno',
[…]
```

Surprisingly, when I call the device 'チルno', that gets respected:

```
    name: 'チルno',
```

The Titanium build scripts expect devices to have names and break in particularly spectacular and confusing ways when they don't. I'm sure there are others.

Also filed as [TC-5491](https://jira.appcelerator.org/browse/TC-5491).

**Comment by colons** [comment]
@ingo This is, through my own sheer bloody-minded stubbornness, kind of a blocker for me. Have you any idea what's going on?

**Comment by ingo** [comment]
I think @cb1kenobi would be a better person to answer that. Chris?

**Comment by cb1kenobi** [comment]
I was able to reproduce the issue and I fixed it as apart of https://jira.appcelerator.org/browse/TIMOB-18929.

Here's the PR where this fix is: https://github.com/appcelerator/node-ios-device/pull/4/files#diff-a1a2ab8f1e1460112c08a94189d9c4efR34

**Comment by colons** [comment]
Any word on when that's likely to get pushed to the npm version?

**Comment by cb1kenobi** [comment]
@colons Just finished testing the heck out of the latest and published v0.4.1! Go get it!

    npm install node-ios-device

**Comment by colons** [comment]
ヽ( ﾟヮ・)ノ thanks
